app/build.gradle.kts

```diff
@@ -39,7 +42,7 @@ android {
                 srcDir("src/main/lib/powerampapi/poweramp_api_lib/res/")
             }
             jniLibs {
-                srcDir("/work/android-root/lib")
+                srcDir("/tmp/android-root/lib")
```

Comment:
I see, this is supposed to be the outcome of the bash script now. Maybe it doesn't hurt to add a comment above to describe where this is coming from